var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -24,15 +24,16 @@ class Neuron(Package):
     git = "https://github.com/neuronsimulator/nrn"
 
     # Patch which reverts 81a7a39 for numerical compat
-    patch("revert_Import3d_numerical_format.master.patch", when="@develop")
+    patch("revert_Import3d_numerical_format.master.patch", when="@7.8.0c:develop")
```

Comment:
```suggestion
    patch("revert_Import3d_numerical_format.master.patch", when="@7.8.0c:")
```
Since the version range is inclusive, might as well just make this open-ended.